python/lsst/drp/tasks/reprocess_visit_image.py

```diff
@@ -148,7 +148,7 @@ class ReprocessVisitImageConfig(
     )
     remove_initial_photo_calib = pexConfig.Field(
         dtype=bool,
-        default=True,
+        default=False,
```

Comment:
Is this False because default DRP pipelines don't apply the photocalib in CalibrateImageTask?